R/build-reference.R

```diff
@@ -348,7 +348,7 @@ data_reference_topic <- function(topic,
 
   # Sections that contain arbitrary text and need cross-referencing
   out$description <- as_data(tags$tag_description[[1]])
-  out$opengraph <- list(description = strip_html_tags(out$description$contents))
+  out$opengraph <- list(description = str_squish(strip_html_tags(out$description$contents)))
```

Comment:
Isn't the squishing now done by desc?

Reply:
Nope, not in this case since the input here stems from parsing `.Rd` files.